shell/browser/native_window.h

```diff
@@ -397,7 +397,9 @@ class NativeWindow : public base::SupportsUserData,
 
   bool has_frame() const { return has_frame_; }
 
-  bool has_client_frame() const { return has_client_frame_; }
+  [[nodiscard]] constexpr bool has_client_frame() const {
```

Comment:
Isn't constexpr here misleading ? `has_client_frame_` is runtime dependent and being explicit about that would avoid usages in compile time evaluation.

Reply:
updated in 284d25e101